config/lib/express.js

```diff
@@ -40,8 +40,9 @@ module.exports.initLocalVariables = function (app) {
 
   // Passing the request url to environment locals
   app.use(function (req, res, next) {
-    res.locals.host = req.protocol + '://' + req.hostname;
-    res.locals.url = req.protocol + '://' + req.headers.host + req.originalUrl;
+    var domain = config.app.domain || req.headers.hostname;
```

Comment:
@mleanos I just checked the express docs: req.hostname is the same as req.headers.hostname (except when `trust proxy` is on). So I agree, we should switch to `req.hostname`.